superbench/benchmarks/micro_benchmarks/directx_gpu_encoding_latency.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the DirectXGPUEncodingLatency benchmarks."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+def create_nv12_file(file_name, num_frames, width, height):
+    """Create a NV12 file with the specified name, number of frames, width, and height."""
+    import numpy as np
+    # Generate a Y plane of width x height with values from 0-255
+    y_plane = np.random.randint(0, 256, (height, width), dtype=np.uint8)
+    # Generate a UV plane of width x height/2 with values from 0-255
+    uv_plane = np.random.randint(0, 256, (height // 2, width), dtype=np.uint8)
+    # Create the file
+    with open(f'{file_name}.nv12', 'wb') as f:
+        for _ in range(num_frames):
+            # Write the Y plane and UV plane to the file
+            f.write(y_plane.tobytes())
+            f.write(uv_plane.tobytes())
+
+
+class DirectXGPUEncodingLatency(MicroBenchmarkWithInvoke):
+    """The DirectXGPUEncodingLatency benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor."""
+        super().__init__(name, parameters)
+        self._bin_name = 'EncoderLatency.exe'
+        self._test_file = 'test_nv12'
```

Comment:
why not 'test_directx_gpu_encoding_latency.nv12'

Reply:
ok